apps/drain-safe/src/hooks/useServices/index.ts

```diff
@@ -0,0 +1,40 @@
+import { useState, useEffect } from 'react';
+import Web3 from 'web3';
+import { Networks } from '@gnosis.pm/safe-apps-sdk';
+
+import InterfaceRepository from './interfaceRepository';
+import { InterfaceRepo } from './interfaceRepository';
+import { rpcUrlGetterByNetwork } from '../../utils';
+
+export interface Services {
+  web3: Web3 | undefined;
+  interfaceRepo: InterfaceRepo | undefined;
+}
+
+export default function useServices(network: Networks): Services {
+  const [web3, setWeb3] = useState<Web3 | undefined>();
+  const [interfaceRepo, setInterfaceRepo] = useState<InterfaceRepository | undefined>();
```

Comment:
`interfaceRepo` doesn't seem to be used anywhere

Reply:
It seems to be used in that file. Tbh, I just copied the hooks from TX Builder.